docs/source/_templates/theme_variables.jinja

```diff
@@ -6,7 +6,7 @@
   'docs': 'https://lightning-flash.readthedocs.io',
   'twitter': 'https://twitter.com/PyTorchLightnin',
   'discuss': 'https://pytorch-lightning.slack.com',
-  'tutorials': 'https://github.com/PyTorchLightning/lightning-flash/tree/master/flash_notebooks',
+  'tutorials': 'https://github.com/Lightning-AI/tutorials',
```

Comment:
This needs to be updated with the correct URL. Currently, the URL in the description of tutorials repository is incorrect/down (https://github.com/Lightning-AI/tutorials/issues/184).